configure.ac

```diff
@@ -1271,6 +1271,10 @@ AC_INCLUDES_DEFAULT
           AC_DEFINE(HAVE_OLD_GSSMIT, 1,
             [if you have an old MIT Kerberos version, lacking GSS_C_NT_HOSTBASED_SERVICE])
         ])
+        dnl check for credential-store support
+        AC_CHECK_HEADERS(gssapi/gssapi_ext.h)
+        AC_CHECK_LIB(gssapi_krb5, gss_accept_sec_context)
```

Comment:
Yea, it's work on progress, I have to fix it (I'm not even using the HAVE_X properly).
Thanks.